pkg/controller/util.go

```diff
@@ -512,6 +513,9 @@ func getPodsUsingPVCs(c client.Client, namespace string, names sets.String, allo
 
 	var pods []v1.Pod
 	for _, pod := range pl.Items {
+		if pod.Status.Phase == v1.PodSucceeded || pod.Status.Phase == v1.PodFailed {
```

Comment:
Pod phase should never be Failed because we use a podRestartPolicy of onFailure

Reply:
it should handle 'other' pods that are not created by cdi